src/devices/RGBDToPointCloudSensorWrapper/RGBDToPointCloudSensor_nws_ros.h

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (C) 2006-2021 Istituto Italiano di Tecnologia (IIT)
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the
+ * BSD-3-Clause license. See the accompanying LICENSE file for details.
+ */
+
+#ifndef YARP_DEV_RGBDTOPOINTCLOUDSENSOR_NWS_ROS_H
+#define YARP_DEV_RGBDTOPOINTCLOUDSENSOR_NWS_ROS_H
+
+#include <vector>
+#include <iostream>
+#include <string>
+#include <sstream>
+
+#include <yarp/os/Port.h>
+#include <yarp/os/Time.h>
+#include <yarp/os/Stamp.h>
+#include <yarp/os/Network.h>
+#include <yarp/os/Property.h>
+#include <yarp/os/PeriodicThread.h>
+
+#include <yarp/sig/Vector.h>
+
+#include <yarp/dev/IWrapper.h>
+#include <yarp/dev/IMultipleWrapper.h>
+#include <yarp/dev/PolyDriver.h>
+#include <yarp/dev/IRGBDSensor.h>
+#include <yarp/dev/FrameGrabberControlImpl.h>
+
+// ROS stuff
+#include <yarp/os/Node.h>
+#include <yarp/os/Publisher.h>
+#include <yarp/os/Subscriber.h>
+#include <yarp/rosmsg/TickTime.h>
+#include <yarp/rosmsg/sensor_msgs/PointCloud2.h>
+
+#define DEFAULT_THREAD_PERIOD   0.03 // s
```

Comment:
You should use `constexpr` instead of `define` whenever possible.

```suggestion
constexpr double DEFAULT_THREAD_PERIOD = 0.03 // s
```